.github/workflows/flaky.yaml

```diff
@@ -55,12 +55,42 @@ jobs:
           result=$(echo '${{ toJSON(needs) }}' | jq -r .tests.result)
           echo TESTS_RESULT=$result
           echo "TESTS_RESULT=$result" >>"$GITHUB_ENV"
+      - name: download nextest reports
+        uses: actions/download-artifact@v4
+        with:
+          pattern: libtest_run_${{ github.run_number }}-${{ github.run_attempt }}-*
+          merge-multiple: true
+          path: nextest-results
+      - name: create summary report
+        id: make_summary
+        run: |
+          # prevent the glob expression in the loop to match on itself when the dir is empty
+          shopt -s nullglob
+          # to deal with multiline outputs it's recommended to use a random EOF
+          # EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+          EOF=aP51VriWCxNJ1JjvmO9i
```

Comment:
any string that's not valid json should do no?  is leaving the note about random useful?

Reply:
If you want I can remove it. It's a bit esoteric, so I prefer some comment - any comment - over nothing. 

I am not sure any non json is valid. Maybe a better comment would be a reference to the syntax, which explains the why, rather than command, which explains the how

Reply:
done in d94b7b67987f874435dcebc95f5f37043645d426